lib/drab.ex

```diff
@@ -213,6 +213,15 @@ defmodule Drab do
     {:noreply, state}
   end
 
+  # pass any other handle_info to the Commander `handle_info_message/2` callbacks,
+  # which expect `message` and `socket` params (no `state` reply).
+  def handle_info(message, state) do
```

Comment:
I'm not sure this should be in `Drab`, it should probably be in `Drab.Channel`, honestly `Drab` shouldn't be a genserver at all, everything it does and all the information it holds really should be in `Drab.Channel`, but that's a refactor for another day.  ^.^;

For note, Drab.Channel already handles phoenix messages, although it passes them to the client for bouncing back 'through' drab, that should probably be restricted to a specific set of 'types' of topics for such javascript bouncing, so normal phoenix events can be handled differently.  Also, `Drab.Commander` has `subscribe` and `unsubscribe` calls for registering to phoenix pubsub topics through the endpoint via `Drab.Channel`.  I wonder what messages it handles through here actually, I.E. I wonder what the purpose of it is to send only to the JS side, hmm...  This needs to be researched.  Maybe Drab already has this functionality but it could just be 'fleshed out' a little bit...

Reply:
> Maybe Drab already has this functionality but it could just be 'fleshed out' a little bit...

maybe, for now I haven't found a better way to support PubSub natively in a Commander